### DISCUSSION TITLE
What problems have you encountered whilst implementing DevOps?

### DISCUSSION THREAD

**Discussion by tdcox** [cdfoundation/sig-best-practices#5] [General]
Please share your experiences with challenges that you may have seen on the road to a DevOps implementation.

**Comment by tequilarista** [comment]
From the BoF discussion:

- Only one team (or minority of teams) willing to try it, which likely do not represent an end-to-end SDLC (I.e. doesn’t include business side , execs, or complete set of tech teams)
- too much ego with decision makers. Ego often results in strongly beliefs which result in poor non-data driven decision making
- “DevOps” team distinct entity from product dev
- “Can’t we just ‘buy’ DevOps?” — orgs looking for easy technical solutions without addressing cultural element
- overfocusing on full featured deliverables rather than MVP-feedback-iterate loop
